pkg/internal/checkup/trex/configfiles.go

```diff
@@ -31,6 +31,7 @@ const (
 	StreamPyFileName           = "testpmd.py"
 	StreamPeerParamsPyFileName = "testpmd_addr.py"
 	ExecutionScriptName        = "run_trex_daemon"
+	BinDirectory               = "/opt/trex"
```

Comment:
We do not control the /opt/trex folder, and so we may collide with file names already existing, or that will in future trex versions we install. 
Can we use another directory? suggesting "/opt/script".

Reply:
I agree.
Is it OK with you to change it in a follow-up PR (after we will get the checkup working)?